geopandas/geodataframe.py

```diff
@@ -952,7 +952,8 @@ def to_wkb(self, hex=False, **kwargs):
             The default is to return a binary bytes object.
         kwargs
             Additional keyword args will be passed to
-            :func:`pygeos.to_wkb` if pygeos is installed.
+            :func:`pygeos.to_wkb` if pygeos is installed or
+            :func:`shapely.to_wkb` if shapely >= 2 is installed.
```

Comment:
Even though not alphabetical, might be good to consider putting shapely first as we prefer that over pygeos

Reply:
We "prefer" that users use shapely, but geopandas itself will still "prefer" (select) pygeos if both are installed, so that could go in both ways ;) 
But yes, it is good to push people towards shapely, so will switch this around :+1: